client/src/components/share/share-template.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { faTwitter } from '@fortawesome/free-brands-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import type { ShareRedirectProps } from './types';
+
+export const ShareTemplate = ({
+  handleRedirectToTwitter
+}: Pick<ShareRedirectProps, 'handleRedirectToTwitter'>): JSX.Element => {
+  return (
+    <button
```

Comment:
This should be a link not a button since it takes you to Twitter's web page. It also needs an accessible name added using either sr-only text or `aria-label`, probably something like "Share on Twitter". You can also add a `title` attribute with the same string you use for the accessible name.

Reply:
it should look like a button, not be a button

Reply:
Yes, it should be a link, but you can style the link to look like a button.

Reply:
ok i will try to do like that

Reply:
Yes, for accessibility. The general rule is "links take you places, buttons do things.". When a screen reader user hears that it is a link then they will know that when they activate the link they will go to a different page. Also, since this links opens a new window we need to add "opens in new window" onto the end of the name so screen reader users will be prepared for this as well. It would be a little strange to have that wording for a button.

Reply:
Updated